client/models.py

```diff
@@ -29,4 +34,5 @@ def delete_client(client):
         raises:
         ValueError error: if the client is not in DB.
         """
-        Client.objects.get(user_name=client.user_name).delete()
+        # The Client object will be deleted automatically once the User object it references is deleted.
+        User.objects.get(username=self.user_name).delete()
```

Comment:
Are we testing this is being deleted?

Reply:
Yes the normal delete client,Will be delete both user and client at the same time and it works,since user delete function works and built in to users

Reply:
Can you point me to the test?